src/cascadia/TerminalControl/TermControl.h

```diff
@@ -201,6 +201,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         std::optional<wchar_t> _leadingSurrogate;
 
         std::optional<Windows::UI::Xaml::DispatcherTimer> _cursorTimer;
+        Windows::Foundation::Point _hoverPos;

```

Comment:
are we using this _hoverPos anymore?

Reply:
Commented out for now (explanation in above reply)